ipam/http.go

```diff
@@ -83,7 +83,7 @@ func (alloc *Allocator) HandleHTTP(router *mux.Router, defaultSubnet address.CID
 	})
 
 	router.Methods("GET").Path("/consensus").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		alloc.Consense()
+		alloc.Prime()
```

Comment:
`GET /prime` is odd; also note that it doesn't say 'GET /consense'. Am going with `GET /ring`.